incubator/binding-asyncapi.spec/src/main/scripts/io/aklivity/zilla/specs/binding/asyncapi/schema/asyncapi.schema.patch.json

```diff
@@ -41,7 +41,19 @@
                                 "type": "array",
                                 "items":
                                 {
-                                    "type": "string"
+                                    "type": "object",
+                                    "additionalProperties": false,
+                                    "properties":
+                                    {
+                                        "apiId":
+                                        {
+                                            "type": "string"
+                                        },
+                                        "location":
+                                        {
+                                            "type": "string"
+                                        }
```

Comment:
I think we want to make this consistent with `openapi` and `openapi-asyncapi` using `patternProperties` so that the `apiId` is the property key and the location is the property value.